sysl2/util.go

```diff
@@ -1,11 +1,13 @@
 package main
 
 // Set is Not exported
```

Comment:
Misleading comment. If it's not meant to be exported, call it `set` (or `intSet` for clarity and to reduce the likelihood of a name collision).

Reply:
changed to intSet.